includes/admin/post-types/post-tables/class-llms-admin-post-table-certificates.php

```diff
@@ -33,8 +33,12 @@ class LLMS_Admin_Post_Table_Certificates {
 	 */
 	public function __construct() {
 
+		if ( ! llms_is_block_editor_supported_for_certificates() ) {
+			return;
+		}
+
 		$post_types = array( 'llms_certificate', 'llms_my_certificate' );
-		if ( in_array( llms_filter_input( INPUT_GET, 'post_type', FILTER_SANITIZE_STRING ), $post_types, true ) ) {
+		if ( in_array( llms_filter_input( INPUT_GET, 'post_type', FILTER_SANITIZE_FULL_SPECIAL_CHARS ), $post_types, true ) ) {
```

Comment:
```suggestion
		if ( in_array( llms_filter_input( INPUT_GET, 'post_type' ), $post_types, true ) ) {
```
I think we're fine with the default filter here, right? (do not filter anything) we're not going to use that value if not for this comparison.